internal/worker/worker.go

```diff
@@ -422,3 +447,12 @@ func (w *BaseWorker) Stop() error {
 	w.Session.Expire()
 	return nil
 }
+
+func GenerateRandomUint32() (uint32, error) {
```

Comment:
@soumya-codes generating random request id to use it in the storeOP. Is this looks okay @soumya-codes